app/models/form/sales/sections/household.rb

```diff
@@ -7,6 +7,7 @@ def initialize(id, hsh, form)
     @form = form
     @subsections = [
       Form::Sales::Subsections::HouseholdCharacteristics.new(nil, nil, self),
+      Form::Sales::Subsections::HouseholdSituation.new(nil, nil, self),
```

Comment:
status tag isn't updating correctly, tests need to be added for this and the question and page